Add a check result entry to nested forms (#748)
yiisoft/validator: CHANGELOG.md

## 2.1.1 under development

> - Bug #748: Fix hook processing for nested objects that implement `PostValidationHookInterface` when `Nested` rule is used (@DAGpro)

Reviewer: ```suggestion
- Bug #748: Fix hook processing for nested objects that implement `PostValidationHookInterface` when `Nested` rule with
  specified rules is used (@DAGpro)
```

## 2.1.1 under development

> - Bug #748: Fix hook processing for nested objects that implement `PostValidationHookInterface` when `Nested` rule with

Reviewer 2: ```suggestion
- Bug #748: Fix hook processing for nested objects that implement `PostValidationHookInterface` when `Nested` rule is used (@DAGpro)
```

>   rules is used (@DAGpro)

Reviewer 2: ```suggestion
```
